readthedocs/doc_builder/python_environments.py

```diff
@@ -219,27 +219,15 @@ def install_core_requirements(self):
         else:
             requirements.extend(
                 [
-                    self.project.get_feature_value(
-                        Feature.USE_SPHINX_LATEST,
-                        positive="sphinx",
-                        negative="sphinx<2",
-                    ),
-                    # If defaulting to Sphinx 2+, we need to push the latest theme
-                    # release as well. `<0.5.0` is not compatible with Sphinx 2+
-                    self.project.get_feature_value(
-                        Feature.USE_SPHINX_LATEST,
-                        positive="sphinx-rtd-theme",
-                        negative="sphinx-rtd-theme<0.5",
-                    ),
+                    "sphinx",
+                    "sphinx-rtd-theme",
```

Comment:
This will continue to be a good "guard" for us: The **latest** Sphinx version that we install means the one that is allowed by the latest `sphinx-rtd-theme`.